clarity_ext/repository/step_repository.py

```diff
@@ -100,16 +100,6 @@ def _wrap_input_output(self, input_info, output_info, container_repo, process_ty
             raise NotImplementedError(
                 "Generation type {} is not implemented".format(output_gen_type))
 
-        if isinstance(output, ResultFile):
-            output.output_type = Artifact.OUTPUT_TYPE_RESULT_FILE
-        elif isinstance(output, Analyte):
-            output.output_type = Artifact.OUTPUT_TYPE_ANALYTE
-        elif isinstance(output, SharedResultFile):
-            output.output_type = Artifact.OUTPUT_TYPE_SHARED_RESULT_FILE
-        else:
-            raise NotImplementedError(
```

Comment:
To make sure that this retains backwards compatibility, perhaps the `NotImplementedError` should be thrown by the base class that the other classes implement?